bokeh/sphinxext/bokeh_plot.py

```diff
@@ -271,8 +282,12 @@ def _process_script(source, filename, env, js_name, use_relative_paths=False):
     with open(js_path, "w") as f:
         f.write(js)
 
-    return (script, js, js_path, source)
+    return (script, js, js_path, source, c.doc)
```

Comment:
```suggestion
    return (script, js, js_path, source, c.doc.strip())
```
The problem in https://github.com/bokeh/bokeh/pull/11433/files#r672548700 with the docstring summary taking up two lines, is caused by the initial space in the first line. This should fix that problem (so the docstrings can be changed 80 columns) 

As an aside, apparently our convention of an initial space in docstrings is very unusual. I think it makes them much more readable, personally, but maybe we should consider changing them everywhere just to be consistent with the rest of the world (not in this PR)